src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition-duration: $transmission-duration;
```

Comment:
Add property to  transition ![image](https://user-images.githubusercontent.com/70009205/217066376-d17c185b-28c1-4ee1-a04b-13a382b3ab15.png)
Fix everywhere